### PR TITLE
add functions for streaming a file from Data

### PR DESCRIPTION
**Pull request by dborncamp**

### CODE REVIEW

File: pygmdata/pygmdata.py [hunk 1]
```diff
@@ -274,7 +276,7 @@ def make_directory_tree(self, path, object_policy=None, **kwargs):
 
         self.log.debug("New file under parent OID: {}".format(oid))
         body = {
-            "action": "C",
+            "action": "U",
```
Comment (hiromis) [inline pygmdata/pygmdata.py:279]: This works even if it's C. I am not 100% sure why but I like `C`

File: pygmdata/pygmdata.py [hunk 2]
```diff
@@ -119,6 +120,7 @@ def create_meta(self, data_filename, object_policy=None,
             - security - The security tag of the given file. If not supplied
             it will keep what is already there or it will use the field
             from the parent if creating a new file.
+            - mimetype - The
```
Comment (hiromis) [inline pygmdata/pygmdata.py:123]: The .... 😂

File: pygmdata/pygmdata.py (hunk 1 above)
Comment (dborncamp) [reply pygmdata/pygmdata.py:279]: Agreed, it makes more sense to be `C` since it is created.

Comment (hiromis) [reply pygmdata/pygmdata.py:280]: it still says U

**Pull request merged**